Configuration/Geometry/python/GeometryExtended2023D5_cff.py

```diff
@@ -9,3 +9,5 @@
 from Geometry.HcalCommonData.hcalDDDSimConstants_cfi import *
 from Geometry.HGCalCommonData.hgcalV6ParametersInitialization_cfi import *
 from Geometry.HGCalCommonData.hgcalV6NumberingInitialization_cfi import *
+from Geometry.HGCalCommonData.fastTimeParametersInitialization_cfi import *
```

Comment:
these lines should be added as the "reco" entry for I2 in timingDict in dict2023Geometry.py (no direct editing of 2023 geometry configs)